pkg/gadgets/trace/network/tracer/tracer.go

```diff
@@ -43,6 +44,7 @@ type link struct {
 
 	sockFd int
 
+	// Identifies the belonging network namespace of the event
 	containerQuark uint64
```

Comment:
```suggestion
	// Identifies the belonging network namespace of the event
	netnsid uint64
```
would that be a better variable name with this PR?